internal/ledger/jsonl.go

```
@@ -0,0 +1,234 @@
+	path string
+}
+
+// NewJSONLStore 打开（或创建）path 上的 JSONL 事件库并完成崩溃恢复。
+func NewJSONLStore(path string) (*JSONLStore, error) {
+	f, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE, 0o600)
+	if err != nil {
+		return nil, fmt.Errorf("ledger: 打开事件库 %s: %w", path, err)
+	}
+	maxSeq, cleanEnd, err := scanFile(f)
+	if err != nil {
+		f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)